pennylane/operation.py

```diff
@@ -900,6 +910,48 @@ def matrix(self):
         # over the defined wires.
         return functools.reduce(np.kron, U_list)
 
+    def prune(self):
+        """Returns a pruned tensor product of observables by removing :class:`~.Identity` instances from
+        the observables building up the :class:`~.Tensor`.
+
+        The return_type attribute is preserved while pruning.
+
+        If the tensor product only contains one observable, then this observable instance is
+        returned.
+
+        Note that this way this method might return an instance that is not a :class:`~.Tensor`
+        instance.
+
+        **Example:**
+
+        1.
```

Comment:
ReST might not render this correctly (it may automatically indent it as an itemized list).

Might be best to indent the code-blocks by 3 spaces, and see what it looks like.